python_modules/libraries/dagster-shared/dagster_shared/telemetry/__init__.py

```diff
@@ -35,12 +38,7 @@
 
 
 def _get_dagster_module_version() -> Optional[str]:
-    try:
-        from dagster.version import __version__ as dagster_module_version
-
-        return dagster_module_version
-    except ImportError:
-        return None
+    return core_version_from_library_version(__version__)
```

Comment:
leave a comment as to whats going on here, likely confusing without context i think

Reply:
we can just move this to the callsite and rm this fn https://github.com/dagster-io/dagster/blob/master/python_modules/libraries/dagster-shared/dagster_shared/telemetry/__init__.py#L197